src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -155,6 +155,8 @@ public bool LayoutSet
 
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
+        // GetViewStyle() is not public.
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1721: Property names should not match get methods")]
```

Comment:
Please remove View.GetViewStyle().
Replace all code that uses GetViewStyle() to CreateViewStyle().
And then you may no need this `SuppressMessage`.